uw-frame-components/portal/messages/partials/announcement-popup-template.html

```diff
@@ -11,8 +11,8 @@
   <md-dialog-content>
     <div class="md-dialog-content features-popup" layout="column" layout-align="center center">
       <img ng-src="{{ latestAnnouncement.featureImageUrl }}" aria-describedby="featureContent" hide-xs>
-      <p id="featureContent" class="sr-only" hide-xs>{{ latestAnnouncement.description }}</p>
-      <p hide-gt-xs>{{ latestAnnouncement.description }}</p>
+      <p id="featureContent" class="sr-only">{{ latestAnnouncement.description }}</p>
+      <p>{{ latestAnnouncement.description }}</p>
```

Comment:
Could this `<p>` pick up the `id="featureContent" and the previous line's `<p` currently with that id simply be deleted?